scripts/setup/create_multinode.sh

```diff
@@ -53,6 +53,16 @@ if [ $PODS_PER_NODE -gt 1022 ]; then
     exit 1
 fi
 
+if [ "$#" -lt $CONTROL_PLANE_REPLICAS ]; then
+    echo "Not enough nodes to set up the requested number of control plane replicas."
```

Comment:
Need to reflect this in the docs/loader.md. I suppose, first 1, 3, or 5 nodes from the argument list will be used for control-plane.

Also, how it would play along with labeling the nodes for master/monitoring/worker?

Reply:
Added to `docs/loader.md`. Labeling is handled automatically by `kubeadm`. All control plane nodes get their proper role when listed with `kubectl get nodes`.

Reply:
No, I meant the label.sh that we use. It has labels for master, monitoring (which we use for load generator) and worker.

Reply:
This should work because all control replicas get `control-plane` tag.
![image](https://github.com/vhive-serverless/invitro/assets/16463659/27af5d0c-3b0c-4536-8f94-0c21f7c74b3d)

Reply:
Loader node will be mislabelled in case of more than 1 master node, since in current implementation we are using the second node that we pass to the script for that. With several master nodes, we will change that behaviour, we will need to change the $LOADER_NODE_NAME to appropriate one here.

Reply:
Fixed. 
![image](https://github.com/vhive-serverless/invitro/assets/16463659/fffcfd4e-d793-47d5-8855-7896aeb5b239)